### ISSUE TITLE
Font asset files specified in the CSS of the page is reporting a 404 Not Found error

### ISSUE THREAD

**Issue by joesb** [cabinetoffice/national-leadership-centre-drupal#50]
**Describe the bug**
Font asset files specified in the CSS of the page is reporting a 404 Not Found error

Errors appearing in logs, reporting 404 Not Found on the URL:
https://connect.nationalleadership.gov.uk/themes/custom/contrib/govuk_design_system/assets/fonts/bold-a2452cb66f-v1.woff2?q=themes/custom/contrib/govuk_design_system/assets/fonts/bold-a2452cb66f-v1.woff2

This has an incorrect element in the URL path — `themes/custom/contrib` — the `custom` part suggests the compiles SASS is not traversing back far enough to the `/themes` web root directory.

I think the problem is at 
https://github.com/cabinetoffice/national-leadership-centre-drupal/blob/master/web/themes/custom/bevan/sass/styles.scss#L3

It needs one more `../` in the `$govuk-assets-path` path.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to any page on https://connect.nationalleadership.gov.uk
2. Open the browser developer tools > Network tab
3. Reload the page
4. See error as 404 on missing files

**Expected behaviour**
Font assess should be loaded without 404 errors.

**Screenshots**
<img width="1573" alt="Screenshot 2019-10-18 at 15 26 49" src="https://user-images.githubusercontent.com/518081/67102786-f7cb8780-f1bb-11e9-8192-e6d39ab7cfe7.png">

**Comment by joesb** [comment]
I would fix this, but I don't know how to recompile the SASS file in the Bevan theme.

I tried 

```shell
$ sass --watch sass/styles.scss css/styles.css
```

But got this error:

```shell
Traceback (most recent call last):
	2: from /usr/local/bin/sass:22:in `<main>'
	1: from /Users/joebaker/.rvm/rubies/ruby-2.6.0/lib/ruby/2.6.0/rubygems.rb:262:in `bin_path'
/Users/joebaker/.rvm/rubies/ruby-2.6.0/lib/ruby/2.6.0/rubygems.rb:283:in `find_spec_for_exe': can't find gem sass (>= 0.a) with executable sass (Gem::GemNotFoundException)
```

**Comment by mikebell** [comment]
Ready for review.

**Comment by mikebell** [comment]
For future reference to rebuild the theme you need to cd to the bevan directory and run:

    npm install
    ./node_modules/.bin/gulp build

**Comment by joesb** [comment]
Fixed by #51
